src/main/java/hudson/remoting/DurationSecondsOptionHandler.java

```diff
@@ -0,0 +1,38 @@
+package hudson.remoting;
+
+import java.time.Duration;
+import org.kohsuke.args4j.CmdLineException;
+import org.kohsuke.args4j.CmdLineParser;
+import org.kohsuke.args4j.OptionDef;
+import org.kohsuke.args4j.spi.OptionHandler;
+import org.kohsuke.args4j.spi.Parameters;
+import org.kohsuke.args4j.spi.Setter;
+
+/**
+ * Parses a positive number of seconds into a {@link Duration}.
```

Comment:
If we are bothering to introduce a custom option handler, why not make it friendlier and accept `3h`, `1d`, etc.? Is there some official format for concise duration strings?

Reply:
spring boot has a utility for this we could copy
https://github.com/spring-projects/spring-boot/blob/main/spring-boot-project/spring-boot/src/main/java/org/springframework/boot/convert/DurationStyle.java